src/UI/Factory.php

```diff
@@ -336,4 +336,23 @@ public function panel();
 	 **/
 	public function modal();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     A divider marks a thematic change in a sequence of other components.
+	 *   composition: >
+	 *     A divider consists of a horizontal ruler which may be accompanied by a label.
+	 *
+	 * rules:
+	 *   usage:
+	 *     1: >
+	 *       Dividers MUST only be used in in container components that render
```

Comment:
typo with double `in in`. 

I think the rule needs some improvement since i think it's currently to weak and I as a developer do not see that I should not use it vor horizontal listings.